tests/features/steps/steps.py

```diff
@@ -34,6 +35,48 @@ def step_impl_send_client(context, model_name, method_name):
     context.response = method(**kwargs)
     store_last_item(context, model_name)
 
+@when('we upload a binary file with metadata.{model_name}.{method_name}')
+def step_impl_when_upload_with_metadata(context, model_name, method_name):
+    client = get_client(context)
+
+    try:
+        model = getattr(client, model_name)
+    except AttributeError:
+        assert False, 'client.{} is not registered with the client'.format(model_name)
+
+    try:
+        method = getattr(model, method_name)
+    except AttributeError:
+        assert False, 'client.{}.{} is not registered with the client'.format(model_name, method_name)
+
+    kwargs = {} if not context.text else json.loads(apply_placeholders(context, context.text))
+    files = {}
+    if method_name == 'create':
+        files['binary'] = open('tests/fixtures/file_example-jpg.jpg', 'rb')
```

Comment:
Instead of hard-coding the filename, what if we were to prefix the provided filename with the path to the fixtures folder. i.e.:
```python
filepath = 'tests/fixtures/{}'.format(kwargs['docs'][0]['filename'])
files = {'binary': open(filepath, 'rb')}
```

Reply:
resolved